### DISCUSSION TITLE
Github Copilot

### DISCUSSION THREAD

**Discussion by Giash1** [community/community#88768] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi,
I am am a ctive student and have github pro access. But I can't use github copilot and it shows that "[ERROR] [default] [2024-01-19T00:14:48.131Z] Extension activation failed: "Timed out waiting for authentication provider to register". Would you like to help me.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Giash1 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
